tests/imap-detection/test.yaml

```diff
@@ -0,0 +1,36 @@
+requires:
+  files:
+    - src/app-layer-imap.c
```

Comment:
I do not think we should have this requirement.

So, CI will be red until suricata PR gets merged...

And so, a renaming of this file would still run the test in the future...